python/common/instrument.py

```diff
@@ -0,0 +1,88 @@
+from functools import wraps
+from spacetime_local.IApplication import IApplication
+from spacetime_local.IFrame import IFrame
+from __builtin__ import staticmethod
```

Comment:
Not sure why this is there. you arent using it anywhere, and shouldn't it be already builtin...